contributing.md

```diff
@@ -0,0 +1,35 @@
+# How to Contribute and development guidelines
+If you want to get involved in this project you have two general ways to do so.
+
+## Creating Issues
+The first (and easiest) way to contribute is by opening issues for bugs you encountered or new features/enhancements you would like to see implemented.
+
+Your issue should contain these elements:
+- a short and clear description of the issue as title (i.e. "Add the ability to downvote movies")
+- a longer explanation why you would like that feature or how the bug occurred (if applicable with some screenshots)
+
+Some nice to haves for us:
+- an idea how to implement this feature maybe with some references to other projects
+
+## Contributing to the development
+The second way to contribute is to implement/fix specific issues yourself and posting a pull request to this project.
+
+### Getting started with development
+To get your development environment started make sure to have golang installed and up to date.
+After forking this repository and cloning your fork change into the `MoviePolls` folder and you will find the `Makefile` of this project.
+To build the project just execute the `Makefile` with `make`. A new folder `bin` will be created with an executable file called `server`.
+
+Before executing the resulting file you have to create the folder `MoviePolls/db` and within this new folder an empty json file `data.json` (the file is not completely empty but contains `{}`). If you do not create that file beforehand the server will not start.
```

Comment:
This is incorrect.  Just the folder needs to exist, the json file will get created if it's not there (this folder should probably be added to the `Makefile` in a separate commit).

Reply:
Alright, my bad - i was fairly sure that the json file was necessary